embiggen/sequences/link_prediction_sequence.py

```diff
@@ -79,7 +81,7 @@ def __getitem__(self, idx: int) -> Tuple[np.ndarray, np.ndarray]:
         Return Tuple containing X and Y numpy arrays corresponding to given batch index.
         """
         edges, labels = self._graph.link_prediction(
-            idx + self.elapsed_epochs,
+            self._seed + idx + self.elapsed_epochs,
             batch_size=self.batch_size,
             negative_samples=self._negative_samples,
             graph_to_avoid=self._graph_to_avoid,
```

Comment:
Spelling errors
-- (first line) Keras Sequence for running Neural Netwok on graph link prediction. => "Network"
--  Index corresponding to batch to be rendered. => "rendered" is not quite right here, "to be returned" is better

Reply:
Fixed both.